scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -128,20 +128,20 @@
 
     private static final String SESSION_ID_PARAMETER_NAME = "sessionId";
 
+    private static final String JOB_ID_PARAMETER_NAME = "jobId";
+
     private static final String ERROR_MESSAGE = "errorMessage";
 
     private static final String JSON_ERROR = "JSON parse ERROR";
 
     private static final String ERROR_MESSAGE_REGEX = "\"errorMessage\":\"(.*)\",\"stackTrace\"";
 
-    private final int widthWindows = 600;
+    private static final int WINDOW_WIDTH = 600;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)